docs/README.md

```diff
@@ -18,11 +18,17 @@ Set up and configure the main application.
 
 ## Luigi Client
 
-Use the Luigi Client API documentation to learn more about the functions and parameters you can use to communicate with the core application to define lifecycle, links, and the appearance of the application.
+Once you have configured Luigi Core, you can look at the Luigi Client API documentation. Luigi Client enables communication between your main application and a micro frontend. Learn more about the functions and parameters you can use to define lifecycles, links, and the appearance of the application.
 
+* [Communication](communication.md) describes how to configure features related to communication between Luigi Core and Client.
 * [Lifecycle](luigi-client-api.md#lifecycle) gives you details on the lifecycle of listeners, navigation nodes, and Event data.
 * [Link Manager](luigi-client-api.md#linkmanager) allows you to navigate to a specific route. 
-* [UX Manager](luigi-client-api.md#uxmanager) helps you to manage the appearance features in Luigi, such as the behavior of backdrop or loading indictors.
+* [UX Manager](luigi-client-api.md#uxmanager) helps you manage the appearance features in Luigi, such as the behavior of backdrop or loading indictors.
 
-## Common
-* [Communication](communication.md) describes how to configure features related to communication between Luigi Core and Client.
+## Examples
+
+Follow the Luigi [application examples](core/examples) for an in-depth look at Luigi capabilities. 
+
+## Development Guidelines
```

Comment:
```suggestion
## Development guidelines
```
I would use lowercase for the second word - https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#headings